packages/session-replay-browser/src/session-replay.ts

```diff
@@ -233,7 +233,7 @@ export class SessionReplay implements AmplitudeSessionReplay {
 
   getShouldRecord(ignoreFocus = false) {
     if (!this.identifiers || !this.config || !this.identifiers.sessionId) {
-      this.loggerProvider.error(`Session is not being recorded due to lack of config, please call sessionReplay.init.`);
+      this.loggerProvider.warn(`Session is not being recorded due to lack of config, please call sessionReplay.init.`);
```

Comment:
Why the warn here instead of the error?

Reply:
Because it's possible, especially when used as a standalone, to setup the SR SDK without a session id, and add the session id later, at which point recording starts. In this case, it's not an error to use it that way, so this log seemed confusing as an error